docs/mobile/development/configuration.mdx

```diff
@@ -7,6 +7,51 @@ import TabItem from '@theme/TabItem'
 
 # Configuration
 
+Currently, Tauri Mobile support is only available on the `2.0.0` alpha tracks, so the main tauri build tools will not produce a project that supports it. You'll need to update a few dependencies to get setup
+
+Make sure to update both NPM and Cargo dependencies to the 2.0.0-alpha.0 release. You can update the NPM dependencies as follows:
+
+> Note: Alpha builds are updated regularly while this documentation may fall behind. To see the newest version numbers, check the [Tauri Releases on GitHub]
+
+<Tabs groupId="package-manager">
+  <TabItem value="npm">
+
+```shell
+npm install @tauri-apps/cli@next @tauri-apps/api@next
+```
+
+  </TabItem>
+  <TabItem value="Yarn Classic">
+
+```shell
+yarn upgrade @tauri-apps/cli@next @tauri-apps/api@next
+```
+
+  </TabItem>
+  <TabItem value="Yarn Berry">
+
+```shell
+yarn up @tauri-apps/cli@next @tauri-apps/api@next
+```
+
+  </TabItem>
+  <TabItem value="pnpm">
+
+```shell
+pnpm update @tauri-apps/cli@next @tauri-apps/api@next
+```
+
+  </TabItem>
+</Tabs>
+
+To update the Cargo dependencies, run the following in the `src-tauri` folder:
+
+```shell
+cargo add tauri@2.0.0-alpha.0
+cargo add tauri-build@2.0.0-alpha.0 --build
+cargo install tauri-cli --version "^2.0.0-alpha"
```

Comment:
imo this should either be part of the tabber, or separate with a note that it's not required for everyone, only those that use `cargo tauri x` - what do you think?

Reply:
Is this step not still required for someone that has made a project with create-tauri-app today? Don't they have to do both the javascript and cargo updates?

Reply:
oh damn, the preview window is not as clear as i expected it to be. my comment was only about the tauri-cli installation, not tauri and tauri-build, sorry.

Reply:
ah, yes, understood